xbmc/utils/SysfsUtils.h

```diff
@@ -13,10 +13,11 @@
 class SysfsUtils
 {
 public:
-  static int SetString(const std::string& path, const std::string& valstr);
-  static int GetString(const std::string& path, std::string& valstr);
-  static int SetInt(const std::string& path, const int val);
-  static int GetInt(const std::string& path, int& val);
+  static bool SetString(const std::string& path, const std::string& value);
+  static const std::string GetString(const std::string& path);
```

Comment:
Any reason why you return `const std::string` and not `std::string` ?
Same for the `const int` below.

Reply:
+1; never return `const`